src/schema/v2/homeView/index.ts

```diff
@@ -117,21 +127,45 @@ const SectionType = new GraphQLUnionType({
   types: [ArtworksRailSectionType, ArtistsRailSectionType],
 })
 
-// a list (TODO: connection) of sections
-
 const Sections: GraphQLFieldConfig<void, ResolverContext> = {
   type: GraphQLNonNull(GraphQLList(GraphQLNonNull(SectionType))),
   description: "A list of sections on the home view",
   resolve: stubDataResolver,
 }
 
+const SectionsConnectionType = connectionWithCursorInfo({
+  name: "Sections",
+  nodeType: SectionType,
+}).connectionType
+
+const SectionsConnection: GraphQLFieldConfig<any, ResolverContext> = {
+  type: SectionsConnectionType,
+  args: pageable({}),
+  resolve: async (_parent, args, _context, _info) => {
+    const { page, size, offset } = convertConnectionArgsToGravityArgs(args)
+
+    const totalCount = STUB_SECTIONS.length
+    const data = STUB_SECTIONS.slice(offset, offset + size)
```

Comment:
I have a suspicion that it won't work with reverse pagination, I will confirm later

Reply:
Meaning with the `before` arg? Seems fine for now